man/systemd.network.xml

```diff
@@ -244,6 +244,38 @@
           if <literal>RequiredForOnline=no</literal>.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>AdministrativeState=</varname></term>
+        <listitem>
+          <para>Takes a string indicating how networkd should manage the link administrative state.
+          Specifically, this controls how networkd controls the network device's <literal>IFF_UP</literal>
+          flag, which is sometimes controlled by system administrators by running e.g.,
+          <command>ip set dev eth0 up</command> or <command>ip set dev eth0 down</command>.</para>
+
+          <para>Takes one of <literal>start-on</literal>, <literal>always-on</literal>,
+          <literal>manual</literal>, <literal>always-off</literal>, or <literal>start-off</literal>.
+          When <literal>manual</literal>, networkd will not change the administrative state at all;
+          the system administrator must bring the interface administratively up or down, as desired.
+          When <literal>start-on</literal> or <literal>start-off</literal>, networkd will set the
+          administrative state on or off, respectively, when networkd starts, but it will not change
+          the administrative state after that. When <literal>always-on</literal> or
+          <literal>always-off</literal>, networkd will set the administrative state on or off,
+          respectively, when networkd starts as well as any time networkd detects a change in
+          administrative state.</para>
+
+          <para>Note if <literal>always-on</literal> or <literal>always-off</literal> are used,
+          the administrative state can still be manually changed, but networkd will always
+          immediately set the administrative state back to its previous state.  This state bounce
```

Comment:
previous state → state configured with this option

Reply:
"state bounce"? Is that established terminology? I at least (as a non-native English speaker) don#t know the term... Maybe call this "brief state change"

Reply:
i reworded this to try to make it clearer, and dropped the 'bounce', let me know if it's better